bridge/README.md

```diff
@@ -0,0 +1,96 @@
+# Bridge (Setu)
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Listener](#listener)
+- [Processor](#processor)
+- [How to start bridge](#how-to-start-bridge)
+- [Reset](#reset)
+- [Common Issues (FAQ)](#common-issues-faq)
+
+## Overview
+
+Bridge module is responsible for listening to multiple chains and processing the events emitted by them.It converts the emitted data into heimdall messages and send them to the heimdall chain. There are `listener` and `processor` components in the bridge module which are responsible for listening and processing the events respectively as per their module. For example `listner/rootchain.go` is responsible for listening to events coming from rootchain OR L1 i.e Ethereum chain in our case and `listner/maticchain.go` is responsible for listening to events coming from maticchain or L2 i.e Bor chain in our case.
```

Comment:
should we use the term L2? Or find some alternative?

Reply:
L2 is fine, Vitalik just used that and Polygon PoS was mentioned as an L2 there.